include/deal.II/hp/fe_values.h

```diff
@@ -83,6 +84,34 @@ namespace hp
       const QCollection<q_dim> &                              q_collection,
       const UpdateFlags                                       update_flags);
 
+    /**
+     * For timing purposes it may be useful to create all required FE*Values
+     * objects in advance.
```

Comment:
Can you reference at this place the discussion of the "lazy creation" of these objects in the general class documentation? You might also want to mention over there that one can bypass the lazy creation by using this function.